README.md

```diff
@@ -1,6 +1,6 @@
 # eslint-plugin-ban [![Build Status](https://travis-ci.org/remithomas/eslint-plugin-ban.svg?branch=master)](https://travis-ci.org/remithomas/eslint-plugin-ban) [![npm version](https://img.shields.io/npm/v/eslint-plugin-ban.svg?style=flat-square)](https://www.npmjs.com/package/eslint-plugin-ban)
 
-> Allows you to bannish some methods or functions.. Inspired by [tslint ban rule](https://palantir.github.io/tslint/rules/ban/)
+> Allows you to bannish some methods or functions. Inspired by [tslint ban rule](https://palantir.github.io/tslint/rules/ban/)
 
 ## Installation
 
```

Comment:
We may want to display the current version of eslint to use, or something like that.

```md
Works with eslint >=9 for older version please use `eslint-plugin-ban` version `1.6.0`
```